Balance.xcodeproj/project.pbxproj

```diff
@@ -276,7 +272,7 @@
 		653A254F283387FE005D4D48 /* Balance */ = {
 			isa = PBXGroup;
 			children = (
-				4ACC786529AC39A40048D1D1 /* HeaderMenu.swift */,
```

Comment:
I think this should contain other files:

children = (
				2720083329A7E4C20052908D /* BalanceDiaryFeature */,
				8900585629AC0DDA005339D9 /* BalanceDebug.entitlements */,
				2761584E29A662DE006C7AC1 /* Music */,
				653A2550283387FE005D4D48 /* Balance.swift */,
				2F5E32BC297E05EA003432F8 /* BalanceAppDelegate.swift */,
				2F4E23822989D51F0013F3D9 /* BalanceAppTestingSetup.swift */,
				2FC975A72978F11A00BA99FE /* Home.swift */,
				2FC9759D2978E30800BA99FE /* Supporting Files */,
				2761584429A64178006C7AC1 /* Balance-Bridging-Header.h */,
				2761584529A6448C006C7AC1 /* SceneDelegate.swift */,
			);